.gitignore

```diff
@@ -14,3 +14,7 @@
 tsconfig.tsbuildinfo
 .env.local
 tmp
+node_modules
+.yarn/cache/*
```

Comment:
We're using zero installs ( https://yarnpkg.com/features/zero-installs ) so don't need to ignore these:

`.yarn/cache/*`
`yarn.lock`

I think something in your configuration may have used `npm install` which doesn't work so well with yarn...

If it becomes a recurring issue we can make the project package manager agnostic.

Reply:
I ignored `.yarn/cache/*` and `yarn.lock` because they were modified due to yarn updates, which i left out for you to review, and also after running `yarn remove forever`, it was the reason for 300+ file changes in my previous commits, so i chose to ignore them because once a user runs `yarn install`, `yarn.lock` is auto generated.

I installed pm2 using npm because yarn couldn't, i should have mentioned that, but pm2 installation is still covered in README.md, otherwise nothing else is `npm install`, just pm2